prog/dftb+/lib_reks/reksgrad.F90

```diff
@@ -571,14 +572,14 @@ subroutine getHxcKernel(iSquare, getAtomIndex, getDenseAO, over, overSqr, &
     !> long-range gamma integrals in AO basis
     real(dp), allocatable, intent(in) :: LrGammaAO(:,:)
 
-    !> Whether to run a range separated calculation
-    logical, intent(in) :: tRangeSep
-
     !> Algorithms to calculate analytic gradients
     integer, intent(in) :: Glevel
 
-    !> Memory level used in calculation of gradient
-    integer, intent(in) :: Mlevel
+    !> Save 'A' and 'Hxc' to memory in gradient calculation
+    logical, intent(in) :: tSaveMem
+
+    !> Whether to run a range separated calculation
+    logical, intent(in) :: tRangeSep
```

Comment:
Can you rename tRangeSep to isRangeSep. From #392 onwards TRangeSep is now a type name elsewhere.

Reply:
Done with commit 8467c28. Please see this commit.
Here, there are remaining parts using `tRangeSep` in main.F90, mainio.F90, initprogram.F90 files.
At the moment, I do not modify the parts that are not related to REKS.